frameworks/cassandra/build-cassandra-bin.bash

```diff
@@ -0,0 +1,167 @@
+CASSANDRA_STOCK_IMAGE="${CASSANDRA_DIST_NAME}-bin.tar.gz"
+CASSANDRA_CUSTOM_IMAGE="${CASSANDRA_DIST_NAME}-bin-dcos.tar.gz"
+CASSANDRA_STOCK_IMAGE_DOWNLOAD_URL="https://archive.apache.org/dist/cassandra/${CASSANDRA_VERSION}/${CASSANDRA_STOCK_IMAGE}"
+
+function _sha1sum {
+    # Try 'sha1sum' (Linux) with fallback to 'shasum' (OSX)
+    SHA1SUM_EXE=$(which sha1sum || which shasum)
+    $SHA1SUM_EXE $@
+}
+
+function _download {
+    if [ ! -f "$2" ]; then
+        wget --progress=dot -e dotbytes=1M -O $2 $1
+        _sha1sum $2
+    fi
+}
+
+function _package_github {
+    PROJECT_NAME=$(basename $1) # user/pkg => pkg
+    if [ ! -f "$PROJECT_NAME/$3" ]; then
+        echo "Building $PROJECT_NAME/$3 from $1:$2"
+        if [ ! -d "$PROJECT_NAME" ]; then
+            time git clone "https://github.com/$1" --branch "$2" --single-branch --depth 1  && cd "$PROJECT_NAME"
+        else
+            cd "$PROJECT_NAME"
+        fi
+        time ./gradlew build
+        cd ..
+    fi
+}
+
+# Build seed-provider jar
+echo "##### Build seed-provider jar #####"
+../../gradlew :cassandra-seed-provider:jar
+
+###
+# Go into tmp dir
+###
+
+mkdir -p "cassandra-bin-tmp"
+cd "cassandra-bin-tmp"
+
+###
+# Download and unpack stock cassandra-bin and verify with downloaded sha1 file
+###
+
+_download $CASSANDRA_STOCK_IMAGE_DOWNLOAD_URL.sha1 "${CASSANDRA_STOCK_IMAGE}.sha1"
+_download $CASSANDRA_STOCK_IMAGE_DOWNLOAD_URL "${CASSANDRA_STOCK_IMAGE}"
+if [ "$(_sha1sum $CASSANDRA_STOCK_IMAGE | awk '{print $1}')" != "$(cat ${CASSANDRA_STOCK_IMAGE}.sha1)" ]; then
+    echo "SHA1 MISMATCH: ${CASSANDRA_STOCK_IMAGE}"
+    exit 1
+else
+    echo "Verified: ${CASSANDRA_STOCK_IMAGE}"
+fi
+rm -rf $CASSANDRA_DIST_NAME
+tar xzf $CASSANDRA_STOCK_IMAGE
+
+###
+# Modify stock cassandra config
+###
+
+CONF_DIR="${CASSANDRA_DIST_NAME}/conf"
+_sha1sum "${CONF_DIR}"/* &> confdir-before.txt || true
+
+# Remove default cassandra-rackdc.properties and cassandra-topology.properties.
+# The framework will provide this configuration on-the-fly.
+
+echo "##### Remove cassandra-[rackdc|topology].properties #####"
+
+rm -v "${CONF_DIR}"/cassandra-rackdc.properties
+rm -v "${CONF_DIR}"/cassandra-topology.properties
+
+# Make a copy of cassandra-env.sh named cassandra-env.sh.bak, then comment out any JMX_PORT
+# configuration. This value is customized by the framework.
+
+echo "##### Disable JMX_PORT in cassandra-env.sh #####"
+
+# "JMX_PORT=???" => "#DISABLED FOR DC/OS\n#JMX_PORT=???"
+$SED -i "s/\(^JMX_PORT=.*\)/#DISABLED FOR DC\/OS:\n#\1/g" "${CONF_DIR}"/cassandra-env.sh
+
+_sha1sum "${CONF_DIR}"/* &> confdir-after.txt || true
+
+LIB_DIR="${CASSANDRA_DIST_NAME}/lib"
+_sha1sum "${LIB_DIR}"/*.jar &> libdir-before.txt || true
+
+echo "##### Install custom seed provider #####"
+cp -v ../seed-provider/build/libs/cassandra-seed-provider-${SEED_PROVIDER_VERSION}.jar ${LIB_DIR}
+
+echo "##### Install StatsD metrics support #####"
+
+# StatsD Reporter: custom build with newer version of metrics-core
```

Comment:
This was written roughly a year ago, does this library still need to be custom built or is a stock build available now?